js/bot.js

```diff
@@ -156,7 +200,7 @@ const uitslag = function() {
       delay: 1000,
       content:
         "Bedankt. We zien dat je nog kan verbeteren op " +
-        getLowestScore().thema
+        getLowestScore().thema + " " + getLowestScore().indepth
```

Comment:
Het resultaat van deze code is nu bijvoorbeeld:
> Bedankt. We zien dat je nog kan verbeteren op Mensen Mensen uitleg.

Ziet er een beetje raar uit naar mijn mening